src/main/java/org/jboss/ejb/client/ClusterNodeSelector.java

```diff
@@ -24,15 +24,19 @@
 import java.util.concurrent.ThreadLocalRandom;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import org.jboss.logging.Logger;
 import org.wildfly.common.Assert;
+import org.wildfly.security.manager.WildFlySecurityManager;
 
 /**
  * A selector which selects and returns a node from the available nodes in a cluster. Typical usage of a
  * {@link ClusterNodeSelector} involve load balancing of calls to various nodes in the cluster.
  *
  * @author Jaikiran Pai
+ * @author <a href="mailto:wfink@redhat.com">Wolf Dieter Fink</a>
  */
 public interface ClusterNodeSelector {
+	final static Logger log = Logger.getLogger(ClusterNodeSelector.class);
```

Comment:
Can you use one of the loggers in org.jboss.ejb._private.Logs (MAIN, REMOTING, TXN, INVOCATION)?  It seems MAIN is probably the closest match.

Reply:
@wfink @bmaxwell can you take a look at my comments above?